src/config-schemas/openmrs-esm-patient-registration-schema.ts

```diff
@@ -0,0 +1,34 @@
+import { validators } from '@openmrs/esm-module-config';
+export const esmPatientRegistrationSchema = {
+  personAttributes: {
+    arrayElements: {
+      subTitle: { validators: [validators.isString] },
+      label: { validators: [validators.isString] },
```

Comment:
Please add the `description` field, at least for `subTitle` & `label` (explain where they go), `name` (is it any name we want, or does it have to correspond to something?), `uuid` (UUID of what? The PAT?), `id` (does it matter what this is?).

Reply:
Fixed